doc/3/controllers/document/index.md

```diff
@@ -0,0 +1,55 @@
+---
+code: true
+type: page
+title: create
+description: document:create
+---
+
+# create
+
+Creates a new document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document,
+      final String _id,
+      final ConcurrentHashMap<String, Object> options) throws NotConnectedException, InternalException
```

Comment:
A couple weeks ago, @jenow and I decided to amend our confluence document detailing how optional parameters should be handled by SDKs. Long story short: overloads must be used for methods taking only 1 optional argument, and option objects should only be used when there are 2 or more options.

This has been decided because this greatly simplifies the API exposed by SDKs, as well as our code base.